<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertyTable</class>
 <widget class="QTreeWidget" name="PropertyTable"><widget class="QWidget" name="useless" native="true" />
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="alternatingRowColors">
   <bool>true</bool>
  </property>
  <column>
   <property name="text">
    <string>Attribute</string>
   </property>
  </column>
  <column>
   <property name="text">
    <string>Value</string>
   </property>
  </column>
  <widget class="QLabel" name="itemName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>name</string>
   </property>
  </widget>
  <widget class="QLabel" name="itemType">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="toolTip">
    <string> with module(s) applied on it</string>
   </property>
   <property name="text">
    <string>type</string>
   </property>
  </widget>
  <widget class="QLabel" name="itemFolder">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="text">
    <string>folder</string>
   </property>
  </widget>
  <widget class="QLabel" name="itemFile">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>file</string>
   </property>
  </widget>
  <widget class="QLabel" name="itemModApplied">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> with module(s) applied on it</string>
   </property>
  </widget>
  <widget class="QLabel" name="itemEmpty">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string> empty</string>
   </property>
  </widget>
  <widget class="QLabel" name="itemDeleted">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string> deleted</string>
   </property>
  </widget>
  <widget class="QLabel" name="itemGeneratedBy">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>generated by</string>
   </property>
  </widget>
  <widget class="QLabel" name="itemSize">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>size</string>
   </property>
  </widget>
  <widget class="QLabel" name="itemRelevantMod">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>relevant module(s)</string>
   </property>
  </widget>
  <widget class="QLabel" name="itemChildren">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>children</string>
   </property>
  </widget>
  <widget class="QLabel" name="itemFiles">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>file(s)</string>
   </property>
  </widget>
  <widget class="QLabel" name="itemTotal">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string> totalizing </string>
   </property>
  </widget>
  <widget class="QLabel" name="itemBytes">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string> bytes</string>
   </property>
  </widget>
  <widget class="QLabel" name="itemFolders">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>folder(s)</string>
   </property>
  </widget>
  <widget class="QLabel" name="itemTimes">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>default timestamp</string>
   </property>
  </widget>
  <widget class="QLabel" name="itemExtAttr">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>extended attributes</string>
   </property>
  </widget>
  <widget class="QLabel" name="itemStaAttr">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>static attributes</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
